code/lib/cli/src/generators/AURELIA/index.ts

```diff
@@ -25,7 +25,7 @@ const generator: Generator = async (packageManager, npmOptions, options) => {
   await baseGenerator(packageManager, npmOptions, options, 'aurelia', {
     extraPackages: ['aurelia'],
   });
-  const templateDir = join(getBaseDir(), 'templates', 'aurelia');
+  const templateDir = join(getCliDir(), 'templates', 'aurelia');
```

Comment:
Get rid of AURELIA?

Reply:
Next to the render assets, there seem to be a second set of templates in the cli:

<img width="367" alt="image" src="https://user-images.githubusercontent.com/1035299/198963225-be71568d-19aa-4713-954a-eced08cecda6.png">

I haven't touched those or investigated them. Maybe we can leave those for a different PR. 
(Before I break the universe again 😅 )